ouroboros-network/sim-tests-lib/Test/Ouroboros/Network/PeerSelection/Cardano/PublicRootPeers.hs

```diff
@@ -58,92 +63,96 @@ arbitraryPublicRootPeers peeraddrs = do
 
   return (PublicRootPeers.fromMapAndSet publicConfigPeersMap bootstrapPeers normalLedgerPeers bigLedgerPeers)
 
-instance (Arbitrary peeraddr, Ord peeraddr) =>
-         Arbitrary (PublicRootPeers peeraddr) where
-    arbitrary = do
-        peeraddrs <- arbitrary
-        arbitraryPublicRootPeers peeraddrs
+instance ( Arbitrary peeraddr
+         , Ord peeraddr
+         , extraPeers ~ CardanoExtraPeers peeraddr
```

Comment:
Why not replacing `extraPeers` in the type below?

Reply:
Same as above